### ISSUE TITLE
PHP error on cronjob

### ISSUE THREAD

**Issue by JanKrAppel** [kanboard/plugin-jabber#4]
I'm getting a PHP error when the cronjob for Kanboard is executing. Here's the output:

```PHP
PHP Notice:  Undefined index: task in /var/www/kanboard/plugins/Jabber/Notification/Jabber.php on line 35
PHP Catchable fatal error:  Argument 1 passed to Kanboard\Plugin\Jabber\Notification\Jabber::getMessage() must be of the type array, null given, called in /var/www/kanboard/plugins/Jabber/Notification/Jabber.php on line 39 and defined in /var/www/kanboard/plugins/Jabber/Notification/Jabber.php on line 111
```

I installed the plugin from the plugin manager and configured it as per the Readme. The XMPP server is working.

**Comment by fguillot** [comment]
This bug is now fixed in version 1.0.7 of the plugin.

**Comment by JanKrAppel** [comment]
Yep, works for me now. Thank you!
